tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -75,21 +83,21 @@ def get(self, request, *args, **kwargs):
             subject=None,
             access_token=None,
             options={"verify_nbf": False},
-        )
-        decoded_nonce = decoded_payload["nonce"]
+        )  # pragma: no cover
+        decoded_nonce = decoded_payload["nonce"]  # pragma: no cover
 
-        if not utils.validate_nonce_and_state(
+        if not validate_nonce_and_state(
             decoded_nonce, state, nonce_validator, state_validator
-        ):
+        ):  # pragma: no cover
             msg = "Could not validate nonce and state"
             raise SuspiciousOperation(msg)
 
-        if not decoded_payload["email_verified"]:
+        if not decoded_payload["email_verified"]:  # pragma: no cover
             return Response(
                 {"error": "Unverified email!"}, status=status.HTTP_400_BAD_REQUEST
             )
 
-        try:
+        try:  # pragma: no cover
```

Comment:
It really feels like this try and this exception should be included in coverage, unless i'm not understanding something.